pkgs/development/libraries/gstreamer/bad/default.nix

```diff
@@ -104,6 +104,7 @@ in stdenv.mkDerivation rec {
       sha256 = "0l1f6kqcl04q7w12a2b4qibcvjz6gqhs0csdv2wbvfd6zndpjm6p";
     })
     ./fix_pkgconfig_includedir.patch
+    ./opencv-4.3.patch
```

Comment:
```
(fetchpatch {
  url = "https://gitlab.freedesktop.org/gstreamer/gst-plugins-bad/-/commit/93af23258922d60b7281c380da3db99e358a5711.patch";
  sha256 = "<REPLACE_ME>";
})
```

Reply:
the patch can be now replaced. The commit should stay the same also if you update the PR.

Reply:
I can do it like that, but then I will need to apply two patches:
https://gitlab.freedesktop.org/gstreamer/gst-plugins-bad/-/merge_requests/939
https://gitlab.freedesktop.org/gstreamer/gst-plugins-bad/-/merge_requests/1235

What do you think ?

Reply:
Ok. Leave it like that but add the pull request url above the patch.